common/src/main/resources/docker/docker.bom

```diff
@@ -357,6 +365,46 @@ brooklyn.catalog:
             command: |
               cat ${DOCKER_CERT_PATH}/csr.pem
 
+  - id: docker-engine-with-resilience
```

Comment:
is it ok to leave as `docker.***`? ... as its no longer part of swarm I think this makes more sense